app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -36,13 +41,25 @@
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
 				"doctors", "clinic", "pharmacy", "veterinary",
-				"place_of_worship", "townhall"};
+				"place_of_worship", "townhall", "school", "kindergarten",
+				"shelter", "hospital", "police", "community_centre",
+				"social_facility", "public_building", "university", "college",
```

Comment:
public_building is deprecated

Reply:
Thanks, I'll remove this, if all other points in question are solved.